backend/src/graphql/resolver/UserResolver.test.ts

```diff
@@ -514,18 +514,20 @@ describe('UserResolver', () => {
         await mutate({ mutation: createUser, variables: createUserVariables })
         const emailContact = await UserContact.findOneOrFail({ email: createUserVariables.email })
         emailVerificationCode = emailContact.emailVerificationCode.toString()
-        result = await mutate({
-          mutation: setPassword,
-          variables: { code: emailVerificationCode, password: 'not-valid' },
-        })
       })
 
       afterAll(async () => {
         await cleanDB()
       })
 
-      it('throws an error', () => {
-        expect(result).toEqual(
+      it('throws an error', async () => {
+        jest.clearAllMocks()
```

Comment:
clearing the loggers in a BeforeEach-definition the logger.mock will be cleared for the test in line 539/541 too. This will have the consequence that the logger will not contain any invocation and no message to check for.